openconcept/thermal/tests/test_heat_exchanger.py

```diff
@@ -3,8 +3,7 @@
 import numpy as np
 from openmdao.utils.assert_utils import assert_near_equal, assert_check_partials
 from openmdao.api import IndepVarComp, Group, Problem
-from openconcept.components.heat_exchanger import OffsetStripFinGeometry, OffsetStripFinData, HydraulicDiameterReynoldsNumber, OutletTemperatures, PressureDrop
-from openconcept.components.heat_exchanger import NusseltFromColburnJ, ConvectiveCoefficient, FinEfficiency, UAOverall, NTUMethod, CrossFlowNTUEffectiveness, NTUEffectivenessActualHeatTransfer
+from openconcept.thermal.heat_exchanger import *
```

Comment:
Make sure you are using everything from the heat exchanger api if you are importing all with `*` here.  Typically better to write out each import for readibility.